test/functional/interface_rest.py

```diff
@@ -205,16 +206,16 @@ def run_test(self):
         assert_equal(len(json_obj['utxos']), 1)
 
         # Do some invalid requests
-        self.test_rest_request("/getutxos", http_method='POST', req_type=ReqType.JSON, body='{"checkmempool', status=400, ret_type=RetType.OBJ)
-        self.test_rest_request("/getutxos", http_method='POST', req_type=ReqType.BIN, body='{"checkmempool', status=400, ret_type=RetType.OBJ)
-        self.test_rest_request("/getutxos/checkmempool", http_method='POST', req_type=ReqType.JSON, status=400, ret_type=RetType.OBJ)
+        self.test_rest_request("/getutxos", http_method='POST', req_type=ReqType.JSON, body='{"checkmempool', status=HTTPStatus.BAD_REQUEST, ret_type=RetType.OBJ)
+        self.test_rest_request("/getutxos", http_method='POST', req_type=ReqType.BIN, body='{"checkmempool', status=HTTPStatus.BAD_REQUEST, ret_type=RetType.OBJ)
+        self.test_rest_request("/getutxos/checkmempool", http_method='POST', req_type=ReqType.JSON, status=HTTPStatus.BAD_REQUEST, ret_type=RetType.OBJ)
 
         # Test limits
         long_uri = '/'.join([f"{txid}-{n_}" for n_ in range(20)])
-        self.test_rest_request(f"/getutxos/checkmempool/{long_uri}", http_method='POST', status=400, ret_type=RetType.OBJ)
+        self.test_rest_request(f"/getutxos/checkmempool/{long_uri}", http_method='POST', status=HTTPStatus.BAD_REQUEST, ret_type=RetType.OBJ)
 
         long_uri = '/'.join([f'{txid}-{n_}' for n_ in range(15)])
-        self.test_rest_request(f"/getutxos/checkmempool/{long_uri}", http_method='POST', status=200)
+        self.test_rest_request(f"/getutxos/checkmempool/{long_uri}", http_method='POST', status=HTTPStatus.OK)
```

Comment:
These changes are unrelated to the segmentation fault and at the very least need to be in a separate commit